<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="activityGroupBox2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>521</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Activity</string>
    </property>
    <widget class="QScrollArea" name="displayconfigScrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>521</width>
       <height>80</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>519</width>
        <height>78</height>
       </rect>
      </property>
      <widget class="QPlainTextEdit" name="displayconfigPlainTextEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>521</width>
         <height>79</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="deleteconfigPushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deactivateconfigPushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Deactivate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="activateconfigPushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Activate</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="existingconfigGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Related spare FGCs</string>
    </property>
    <widget class="QComboBox" name="relatedsparesComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>261</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="operationalfgcGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>301</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Operational FGC selection</string>
    </property>
    <widget class="QComboBox" name="operationalfgcComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>261</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="operationalfgcPushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sparefgcPushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
